core/chat/events/events.go

```diff
@@ -5,6 +5,8 @@ import (
 	"regexp"
 	"strings"
 	"time"
+	"sync"
+	"text/template"
```

Comment:
This error just means you should run the go formatter to to re-order the imports. I've never understood why it just doesn't say it's the order that's the problem.